defrag/__main__.py

```diff
@@ -38,9 +41,30 @@ def main() -> None:
                 "Can't have two modules with the same name! Please change one")
 
 
-@app.get("/")
-async def root() -> Dict[str, str]:
-    return {"message": "Hello World"}
+@app.on_event("startup")
+async def startup() -> None:
+    with RedisPool() as conn:
+        conn.flushall()
+    reddit_strategy = CacheStrategy(RedisCacheStrategy(
+        "reddit_default", True, True, 300, None, None), None)
+    twitter_strategy = CacheStrategy(RedisCacheStrategy(
+        "twitter_default", True, True, 300, None, None), None)
+    reddit = ServiceTemplate("reddit", reddit_strategy, None, None, None, None)
+    twitter = ServiceTemplate(
+        "twitter", twitter_strategy, None, None, None, None)
+    ServicesManager.initialize(twitter, reddit)
+    pretty_log("Finished startup with services",
+               str(ServicesManager.services.list_all()))
+
+
+@app.get("/twitter")
```

Comment:
Can you move this to the module file too? See how I do it with bugzilla or the demo module. Because this would mean that if someone wants not to load these modules, they would need to change the main file.

Reply:
What do you mean "if someone wants not to load these modules"? Are we developing a library to be consumer by anyother people than us?